clients/secret/secret_test.go

```diff
@@ -0,0 +1,94 @@
+package secret_test
+
+import (
+	"bytes"
+	"context"
+	"github.com/golang/mock/gomock"
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+	"github.com/influxdata/influx-cli/v2/clients/secret"
+	"github.com/influxdata/influx-cli/v2/config"
+	"github.com/influxdata/influx-cli/v2/internal/mock"
+	"github.com/influxdata/influx-cli/v2/pkg/influxid"
+	"github.com/stretchr/testify/assert"
+	tmock "github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestSecret_List(t *testing.T) {
+	t.Parallel()
+
+	id, err := influxid.IDFromString("1111111111111111")
```

Comment:
Since everything uses this as a `string`, you could skip the `IDFromString` wrapper logic

Reply:
The `clients.OrgParams` struct still requires a type `influxid.ID` instead of String. So I think this is related to #114, but for now I have to stick with the influxid.ID type